tests/algorithms/background/test_gmodel.py

```diff
@@ -31,11 +33,17 @@ def test_simple(dials_data, model, tmpdir):
     reflns_simple.dirpath().ensure(dir=1)
     reflns_g_simple.dirpath().ensure(dir=1)
 
+    exp = load.experiment_list(experiments)
+    panel = exp[0].detector[0]
+    max_trusted = panel.get_trusted_range()[1]
+    panel.set_trusted_range((0, max_trusted))
+    exp.as_json(tmpdir.join("trusted_range_patch.expt"))
```

Comment:
Could you add a brief comment as to why this is necessary? I.e. what was the min_trusted_value in the unpatched experiments?